string color
velocity_cmd limits
---
bool status